docs/api.md

```diff
@@ -1361,7 +1361,7 @@ most.from([1, 2, 3, 4])
 ### switch
 
 ####`stream.switch() -> Stream`
-####`most.switch(stream) -> Stream`
+####`most.switchLatest(stream) -> Stream`
```

Comment:
Using `most.switch(stream)` is still valid for folks who might do `var most = require('most')` or `import * as most from 'most'`.  Should we keep that in the docs, or should we drop it?  If we drop it, that may open up some followup questions:

* Should we deprecate the function name `switch`?
* Should we show `stream.switchLatest()` (i.e. `Stream.prototype.switchLatest`) here in the docs as well?
* If so, for consistency, should we deprecate `Stream.prototype.switch` and go all-in on the name `switchLatest`?

Thoughts?